scripts/masternode.js

```diff
@@ -21,7 +21,9 @@ class Masternode {
 	    if(cMasternodes.length > 0) {
 		return cMasternodes[0];
 	    } else {
-		return "MISSING";
+			const missing= new Object();
+			missing.status="MISSING";
+			return missing;
```

Comment:
This should probably be `return { status: "MISSING" };`

Reply:
fixed